recipes/datreant/meta.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: datreant
+  version: 1.0
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - datreant
+
+about:
+  home: http://datreant.org/
+  license: BSD 3-Clause
```

Comment:
We should include a license file with this as well. Once that is done please use [`license_file`]( http://conda.pydata.org/docs/building/meta-yaml.html#license-file ) to include it in the built package.